jupyter-extension/src/plugins/mount/components/Pipeline/Pipeline.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import {closeIcon} from '@jupyterlab/ui-components';
 import {usePipeline} from './hooks/usePipeline';
-import {PpsContext, PpsMetadata, MountSettings} from '../../types';
+import {PpsContext, PpsMetadata, MountSettings} from 'plugins/mount/types';
```

Comment:
I don't think these changes are relevant to the PR?

Reply:
They're not; it's just a change a pulled out of autosave. I can pull this into its own PR